com/win32com/test/testPyComTest.py

```diff
@@ -886,15 +880,27 @@ def TestQueryInterface(long_lived_server=0, iterations=5):
 
 
 class Tester(win32com.test.util.TestCase):
-    def testVTableInProc(self):
+    def testRegisterInterfacesAfterGencache(self) -> None:
+        # We had a bug where RegisterInterfaces would fail if gencache had
+        # already been run - exercise that here
+        try:
+            RegisterInterfaces("{6BCDCB60-5605-11D0-AE5F-CADD4C000000}", 0, 1, 1)
+        except NotImplementedError:
+            if platform.machine() == "ARM64":
+                raise SkipTest(
+                    "`win32com.universal.RegisterInterfaces` doesn't support ARM64 yet"
+                )
+            raise
```

Comment:
I carved this very specific exception so that if support is added (or attempted) and the error changes or disappears, this will have to be updated accordingly (hopefully by removing the tr-except entirely)